### PR TITLE
[PM-716] add new api for tkm - consent manager

### PR DESCRIPTION
**Pull request by infantesimone**
### List of changes

- add APIs of `tkm-ms-consent-manager` in _Payment Manager_ apim product;
### Motivation and context

The goal of this PR is to expose _[tkm-ms-consent-manager](https://github.com/pagopa/tkm-ms-consent-manager)_ API to `c-star` infrastructure through the apim of the `pagopa platform`. These APIs are protected by subscription keys.

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/env/dev/terraform.tfvars [hunk 1]
```diff
@@ -348,6 +348,9 @@ bpd_hostname       = "api.dev.cstar.pagopa.it"
 cobadge_hostname   = "portal.test.pagopa.gov.it/pmmockserviceapi"
 fesp_hostname      = "portal.test.pagopa.gov.it"
 satispay_hostname  = "portal.test.pagopa.gov.it/pmmockserviceapi"
+cstar_ip_1         = "20.105.180.187"
```
Comment (pasqualespica) [inline src/env/dev/terraform.tfvars:351]: are there ip outbound of lb of aks-cstar ?

File: src/env/uat/terraform.tfvars [hunk 2]
```diff
@@ -75,7 +75,9 @@ app_gateway_deny_paths = [
   "/payment-manager/payment-gateway/*",
   "/pp-restapi-CD/v2/payments/send-payment-*",
   "/pp-restapi-CD/v1/transactions/update-status/*",
-  "/checkout/io-for-node/*"
+  "/checkout/io-for-node/*",
+  "tkm/tkmcardmanager/*",
```
Comment (pasqualespica) [inline src/env/uat/terraform.tfvars:79]: why missing `/` ? it's right ?

**Review by pasqualespica — APPROVED** [review]
lgtm 👍 see comment before merge

File: src/env/dev/terraform.tfvars (hunk 1 above)
Comment (infantesimone) [reply src/env/dev/terraform.tfvars:351]: outbound, I update variable name, thanks

**Pull request merged**